skr/webhook/chart/skr-webhook/templates/deployment.yaml

```diff
@@ -0,0 +1,65 @@
+apiVersion: v1
+kind: Service
+metadata:
+  name: skr-webhook
+  namespace: default
+spec:
+  selector:
+    #selector for deployment labels (key, values)
+    app: skr-webhook
+  ports:
+    - port: {{.Values.service.port}}
+      targetPort: watcher-container-port
+      name: webhook
+
+---
+
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: skr-webhook
+  namespace: default
+  labels:
+    app: skr-webhook
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: skr-webhook
+  template:
+    metadata:
+      labels:
+        app: skr-webhook
+    spec:
+      containers:
+        - name: server
+          env:
+            - name: POD_NAME
+              valueFrom:
+                fieldRef:
+                  fieldPath: metadata.name
+            - name: WEBHOOK_PORT
+              value: {{.Values.service.targetPort}}
+            - name: TLS_ENABLED
+              value: "true"
+            - name: TLS_KEY
+              valueFrom:
+                secretKeyRef:
+                  name: skr-webhook-tls
+                  key: tls.key
+            - name: TLS_CERT
+              valueFrom:
+                secretKeyRef:
+                  name: skr-webhook-tls
+                  key: tls.cert
+            - name: KCP_IP
+              value: "<load-balancer-ip-here>"
+            - name: KCP_PORT
+              value: "80"
+            - name: KCP_CONTRACT
+              value: {{.Values.service.contactVersion}}
+          image: adityabhatia/webhook:0.1.1
```

Comment:
I think this should not bind to your registry, one of the reasons why I think we should go for Kustomize here as we can default it (like "controller" in kubebuilder projects). Alternatively, introduce a helm chart value?

Reply:
https://github.com/kyma-project/kyma-watcher/issues/42